visual-inspection/inspection-frontend/src/components/ExplainInspection.vue

```diff
@@ -27,18 +49,40 @@ export default {
     predictionReady: {
       type: Boolean,
       default: false
+    },
+    isButtonClicked: {
+      type: Boolean,
+      default: false
+    }, 
+    Cls_Acc_List: {
+      type: Object,
+      default: undefined
+    },
+    Cls_Min_Acc :{
+      type: Number,
+      default: 0
```

Comment:
I don't think that these should be `props` of the component. These are just state variables, right? (They should go into `data`, then.)